tests/Fixer/ConstantNotation/NativeConstantInvocationFixerTest.php

```diff
@@ -450,6 +450,35 @@ public function testDoNotFixImportedConstants()
     M_PI,
     M_EULER
 );
+EOT;
+
+        $this->doTest($expected, $input);
+    }
+
+    public function testFixScopedOnly()
```

Comment:
I don't know if it matters anything, but I'm missing a (scoped only) test with no `namespace` keyword at all.

```php
<?php

echo PHP_VERSION . PHP_EOL;
```

Should be untouched